picasso/src/main/java/com/squareup/picasso/Request.java

```diff
@@ -434,6 +436,29 @@ public Builder transform(Transformation transformation) {
       return this;
     }
 
+
+    /**
+     * Set a list of transformations to be applied to the image.  These transformations will
+     * overwrite any existing transformations.
+     * <p>
+     * Use {@link #transform(Transformation)} to add a transformation without overwriting existing
+     * transformations.
+     * <p>
+     * Custom transformations will always be run after the built-in transformations.
+     */
+    public Builder setTransformations(List<Transformation> transformations) {
```

Comment:
Generic should be `? Super Transformation` so someone could pass `List<MyTransformation>`